sirepo/template/srw.py

```diff
@@ -389,20 +389,24 @@ def export_rsopt_config(data, filename):
     v = _rsopt_jinja_context(data.models.exportRsOpt)
 
     fz = pkio.py_path(filename)
-    f = re.sub(r'[^\w\.]+', '-', fz.purebasename).strip('-')
+    f = re.sub(r'[^\w.]+', '-', fz.purebasename).strip('-')
```

Comment:
That's interesting. I didn't know that. I thought you always had to backslash "." in square brackets...